.prettierrc

```
@@ -0,0 +1,3 @@
+{
+   
+  }
```

```suggestion
{
   
  }
``` can you add the valid prettier configuration